src/payment/checkout/Checkout.jsx

```diff
@@ -169,7 +168,7 @@ class Checkout extends React.Component {
             {/* Apple Pay temporarily disabled per REV-927  - https://github.com/openedx/frontend-app-payment/pull/256 */}
           </p>
         </div>
-        {enableStripePaymentProcessor && options.clientSecret ? (
+        {!submissionDisabled && (enableStripePaymentProcessor ? (options.clientSecret && (
```

Comment:
Might be useful to use indents to make the logic here clearer. I'm not sure at a glance which code is in effect when.

Also, I seem to recall that we use `submissionDisabled` for multiple purposes (after submit button is pressed, while coupon is refreshing, etc). Might be worth checking that this isn't changing any user flows on the Cybersource side.

Reply:
fixed indentation, and changed 'submissionDisabled' to 'loaded', tested with both stripe and cybersource successfully.